ast/src/lang/parse.rs

```diff
@@ -1212,6 +1271,18 @@ fn _pick_target_file_from_graph<G: Graph>(target_name: &str, graph: &G) -> Optio
 
     target_file
 }
+fn resolve_import_path(import_path: &str) -> String {
```

Comment:
this probably should be in the trait? since different languages handle things differently. Like go imports from the "package" not from a file